lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -381,6 +393,88 @@ def stop_event_loop(self):
     stop_event_loop.__doc__=FigureCanvasBase.stop_event_loop_default.__doc__
 
 
+_flow = [Gtk.Orientation.HORIZONTAL, Gtk.Orientation.VERTICAL]
+flow_types = ['horizontal', 'vertical']
+
+
+class Window(WindowBase, Gtk.Window):
```

Comment:
Why don't keep the name something like `WindowGTK3` and at the end of the file `Window=WindowGTK3`